eslint-plugin-quintoandar/rules/no-npm-registry.js

```diff
@@ -0,0 +1,41 @@
+const path = require('path');
+
+const npmRegistryPattern = /registry\.npmjs\.org\//g;
+const nexusURL = 'nexus.quintoandar.com.br/repository/npm/';
+
+module.exports = {
+	meta: {
+		type: 'problem',
+		docs: {
+			description: 'Does not allow package-lock.json resolved URLs to be npm\'s registry.',
+			category: 'Infrastructure',
+			recommended: true
+		},
+		fixable: 'code',
+	},
+
+	create: function(context) {
+		let filename = context.getFilename();
+    if (path.basename(filename) !== 'package-lock.json') {
+      return {};
+		}
```

Comment:
the same thing of other comments, please improve the readability of the code

```suggestion
     }
```